starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -52,4 +52,20 @@ public void findAll() {
     List<Inventory> actualInventory = this.inventoryDAO.findAll();
     Assert.assertFalse(actualInventory.isEmpty());
   }
+  
+  /**
+   * Test Create method.
+   */
+  @Test
+  public void create() {
+    Inventory inventory = new Inventory();
+    inventory.setName(NAME);
+    inventory.setProductType(PRODUCT_TYPE);
+    inventory.setId("1454213");
+    Inventory created = inventoryDAO.create(inventory);
+    List<Inventory> actualInventory = this.inventoryDAO.findAll();
+    Assert.assertEquals(actualInventory.get(0).getName(), "Amber");
+    Assert.assertEquals(actualInventory.get(0).getProductType(), "hops");
+    Assert.assertFalse(actualInventory.get(0).getId().equals("1454213"));
```

Comment:
I would recommend using the same constants that were provided at the class level instead of typing out the strings again. This allows you to change them in one place, should you need to.

You can also use an assertNotEquals instead of the assertFalse here.